src/components/Chat/MessageWithLink.js

```diff
@@ -116,7 +116,7 @@ const styles = StyleSheet.create({
     marginVertical: 4
   },
   containerChat: (isMe) => ({
-    backgroundColor: isMe ? COLORS.halfBaked : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.halfBaked : COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comments, please ensure that the change in the background color of the chat container from `COLORS.lightgrey` to `COLORS.gray100` when `isMe` is false is in line with your design guidelines and does not negatively impact usability.
```diff
-    backgroundColor: isMe ? COLORS.halfBaked : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.halfBaked : COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->